src/bors/handlers/trybuild.rs

```diff
@@ -61,13 +62,20 @@ pub(super) async fn command_try_build(
 
     let base_sha = match get_base_sha(&pr_model, parent) {
         Ok(Some(base_sha)) => base_sha,
-        Ok(None) => repo
-            .client
-            .get_branch_sha(&pr.base.name)
-            .await
-            .context(format!("Cannot get SHA for branch {}", pr.base.name))?,
+        Ok(None) => {
+            measure_network_request("get_branch_sha", || async {
```

Comment:
This approach is too disruptive, we need to add `measure_network_request` to many places in the code. Let's put this into the lowest level API possible, which in this case is `GithubRepositoryClient`, to avoid polluting the codebase too much.